docs/img-src/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Mermeid might be a good solution, but we need to either restructure the diagram, or at least remove the arrows going from one end point to itself, it makes the diagram much clearer IMO.